nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -304,18 +338,23 @@ class TfOpIdxMap {
     }
   }
 
-  template <typename T> BidirectionalIndexMap<T> &getIndexMap() {
-    return std::get<BidirectionalIndexMap<T>>(maps);
+  template <typename KT, typename DT = KT>
```

Comment:
Choose one of them ( KT or KeyType ). I prefer KeyType for readability.

Reply:
Updated